### PR TITLE
[ODS-4367] Add command name to net core tools

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1 [hunk 1]
```diff
@@ -34,7 +34,11 @@ param(
 $ErrorActionPreference = 'Stop'
 
 $repositoryNames = @('Ed-Fi-Ods', 'Ed-Fi-ODS-Implementation', 'Ed-Fi-ODS-Tools\Application\EdFi.Ods.AdminApp.Web')
-& "$PSScriptRoot\..\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1" $repositoryNames
+& "$PSScriptRoot\..\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1" $repositoryNames 
+$implementationRepo = Get-Item "$PSScriptRoot\..\..\..\..\.." | Select-Object -Expand Name
+$env:toolsPath = $toolsPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'tools')
+
```
Comment (jamessmckay) [inline logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1:40]: remove xtra line

File: logistics/scripts/modules/tools/ToolsHelper.psm1 [hunk 2]
```diff
@@ -257,20 +257,20 @@ function Invoke-DbDeploy {
...
-        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy.exe"
+        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy"       
     }
-    else {
-        $tool = Get-RepositoryResolvedPath (Join-Path "tools" "EdFi.Db.Deploy.exe")
+    else {                
+        $tool = (Join-Path $env:toolsPath 'EdFi.Db.Deploy')        
     }
-
+
```
Comment (jamessmckay) [inline logistics/scripts/modules/tools/ToolsHelper.psm1:265]: revert extra whitespace

File: logistics/scripts/modules/tools/ToolsHelper.psm1 [hunk 3]
```diff
@@ -257,20 +257,20 @@ function Invoke-DbDeploy {
     $databaseType = $databaseIdLookup[$Database]
 
     if (-not [string]::IsNullOrWhiteSpace($ToolsPath) -and (Test-Path -Path $ToolsPath)) {
-        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy.exe"
+        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy"       
     }
-    else {
-        $tool = Get-RepositoryResolvedPath (Join-Path "tools" "EdFi.Db.Deploy.exe")
+    else {                
+        $tool = (Join-Path $env:toolsPath 'EdFi.Db.Deploy')
```
Comment (jamessmckay) [inline logistics/scripts/modules/tools/ToolsHelper.psm1:263]: revert extra whitespace

File: logistics/scripts/modules/tools/ToolsHelper.psm1 [hunk 4]
```diff
@@ -257,20 +257,20 @@ function Invoke-DbDeploy {
     $databaseType = $databaseIdLookup[$Database]
 
     if (-not [string]::IsNullOrWhiteSpace($ToolsPath) -and (Test-Path -Path $ToolsPath)) {
-        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy.exe"
+        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy"
```
Comment (jamessmckay) [inline logistics/scripts/modules/tools/ToolsHelper.psm1:260]: revert extra whitespace

File: logistics/scripts/modules/tools/ToolsHelper.psm1 [hunk 5]
```diff
@@ -257,20 +257,20 @@ function Invoke-DbDeploy {
     $databaseType = $databaseIdLookup[$Database]
 
     if (-not [string]::IsNullOrWhiteSpace($ToolsPath) -and (Test-Path -Path $ToolsPath)) {
-        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy.exe"
+        $tool = Join-Path $ToolsPath "EdFi.Db.Deploy"       
     }
-    else {
-        $tool = Get-RepositoryResolvedPath (Join-Path "tools" "EdFi.Db.Deploy.exe")
+    else {
```
Comment (jamessmckay) [inline logistics/scripts/modules/tools/ToolsHelper.psm1:262]: revert extra whitespace

File: logistics/scripts/modules/tools/ToolsHelper.psm1 [hunk 6]
```diff
@@ -257,20 +257,20 @@ function Invoke-DbDeploy {
...
+    
     $hasFeatures = ($Features.count -gt 0)
     if ($hasFeatures) {
         & $tool $Verb -e $Engine -d $databaseType -c "$ConnectionString" -f $Features -p $FilePaths | Write-Host
     }
     else {
         & $tool $Verb -e $Engine -d $databaseType -c "$ConnectionString" -p $FilePaths | Write-Host
     }
-
+
```
Comment (jamessmckay) [inline logistics/scripts/modules/tools/ToolsHelper.psm1:273]: revert extra whitespace

File: logistics/scripts/modules/tools/ToolsHelper.psm1 (hunk 2 above)
Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1]: Fixed it . Thanks

File: logistics/scripts/modules/tools/ToolsHelper.psm1 (hunk 3 above)
Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1]: Fixed it . Thanks

File: logistics/scripts/modules/tools/ToolsHelper.psm1 (hunk 4 above)
Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1]: Fixed it . Thanks

File: logistics/scripts/modules/tools/ToolsHelper.psm1 (hunk 5 above)
Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1]: Fixed it . Thanks

File: logistics/scripts/modules/tools/ToolsHelper.psm1 (hunk 6 above)
Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1]: Fixed it . Thanks

File: logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1 (hunk 1 above)
Comment (semalaiappan) [reply logistics/scripts/activities/build/CloudOds/PrepareDatabasesForExport.ps1]: Fixed it . Thanks

**Pull request merged**